### PR TITLE
Vane

### PR DESCRIPTION
**Pull request by vane7192**

### CODE REVIEW

File: src/main/java/vane/MultiplicationTable.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * Created by Vane Gharakhanian on Sep, 2019
+ */
+public class MultiplicationTable {
+    public static void main(String args[]) {
+        Scanner s = new Scanner(System.in);
+        System.out.print("Please Enter number:");
+        int n = s.nextInt();
+        int i = 1;
+        while (i < 10) {
+            System.out.println(n + " x " + i + " = " + n * i);
```
Comment (Smbat888) [inline src/main/java/vane/MultiplicationTable.java:15]: konkret estex, repeat-i qanaky gites, arjer for-ov anel, bayc de senc el klni )

File: src/main/java/vane/PrimeNumber.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package main.java.vane;
+
+import java.util.Scanner;
+
+/**
+ * Created by Vane Gharakhanian on Sep, 2019
+ */
+public class PrimeNumber {
```
Comment (Smbat888) [inline src/main/java/vane/PrimeNumber.java:8]: lava, apres!

File: src/main/java/vane/PrimeNumberHomework.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ * Created by Vane Gharakhanian on Sep, 2019
+ */
+public class PrimeNumberHomework {
+    public static void main(String[] args) {
+        int i, m;
+        boolean isPrime = true;
+        Scanner s = new Scanner(System.in);
+        int n = s.nextInt();
+        m = n / 2;
+        if (n == 0 || n == 1) {
```
Comment (Smbat888) [inline src/main/java/vane/PrimeNumberHomework.java:15]: apres, vor es check-y drel es, te che banic antexyak mardy vor nayi code-y chi jogi vor xia 2-ic sksum )

File: src/main/java/vane/Calculator.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                    System.out.println("Invalid input");
+                } else {
+                    System.out.println(firstNumber / secondNumber);
+                }
+                break;
+            }
+            case '*':
+                System.out.println(firstNumber * secondNumber);
+                break;
+        }
```
Comment (Smbat888) [inline src/main/java/vane/Calculator.java:41]: kakraz arjer defaulti depqn el qnnarkel, u tpel vor sxal operation a mutq arel u while -ov anyndhat asel vor noric mutq ani voch te cragirn exit exni

File: src/main/java/vane/Calculator.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+        switch (operation) {
+            case '+':
+                System.out.println(firstNumber + secondNumber);
+                break;
+            case '-':
+                System.out.println(firstNumber - secondNumber);
+                break;
+            case '/': {
+                if (secondNumber == 0) {
```
Comment (Smbat888) [inline src/main/java/vane/Calculator.java:31]: 0-i checky shat lavna, apres

**Review by Smbat888 — COMMENTED** [review]
lava apres, mi qani hat ban mnac, anes prcnes indznic (es shabatva hamar eli :D)

File: src/main/java/vane/MultiplicationTable.java (hunk 1 above)
Comment (vane7192) [reply src/main/java/vane/MultiplicationTable.java:15]: skzbic for ov ei grel heto pokheci :) kani vor mnacac tegherum elei ogtagorcel for-e mtaceci es meke while ogtagorcem :D

**Pull request merged**